integration/images/image_list.go

```diff
@@ -30,12 +30,13 @@ var imageListFile = flag.String("image-list", "", "The TOML file containing the
 
 // ImageList holds public image references
 type ImageList struct {
-	Alpine           string
-	BusyBox          string
-	Pause            string
-	ResourceConsumer string
-	VolumeCopyUp     string
-	VolumeOwnership  string
+	Alpine               string
+	BusyBox              string
+	Pause                string
+	ResourceConsumer     string
+	VolumeCopyUp         string
+	VolumeOwnership      string
+	ArgsEscapedTestImage string
```

Comment:
No need for TestImage in the field name as we know that these are all images being used for testing

Reply:
```suggestion
	ArgsEscaped string
```